contracts/Core/RewardManager.sol

```diff
@@ -21,7 +21,8 @@ contract RewardManager is Initializable, Constants, RewardManagerParams, IReward
     IBlockManager public blockManager;
     ICollectionManager public collectionManager;
 
-    /** @param stakeManagerAddress The address of the VoteManager contract
+    /**
+     * @param stakeManagerAddress The address of the StakeManager contract
```

Comment:
hey here collection manger also needs to be added as param

Reply:
also please write Natspec for internal functions as well

Reply:
I have not added natspec on internal functions as it is not going to be parsed by the solidity compiler. It is still upto us though whether we want to add natspec to internal functions

Reply:
Lets add them. Will be better for readability and auditing.